posts/2020-03-31-Rust-survey-2019.md

```diff
@@ -0,0 +1,216 @@
+
+These issues are definitely ones we want to address in 2020 and beyond.
+
+## Who is using Rust and what for?
+
+![How long have you worked in Rust](/images/2020-03-RustSurvey/7-How-Long-Worked-With-Rust.png)
+
+We found we have a lot of new folks coming in!
+
+![Size of summed Rust projects](/images/2020-03-RustSurvey/10-Size-Of-Summed-projects.png)
+
+The majority of Rust projects (43%) are 1,000-10,000 lines of code. Rust projects of medium to large size (those totaling over 10k and 100k lines of code respectively) continue to trend higher. They have grown from 8.9% in 2016, to 16% in 2017, to 23% in 2018, to **34%** in 2019.
+
+![How often do you use Rust](/images/2020-03-RustSurvey/11-How-Often-Use-Rust.png)
+
+Rust daily usage has trended slightly upward at 27.63% (it was just under 25% last year and 17.5% on 2017). Users who use Rust daily or weekly has also continued to rend slight upward. This year it was 68.52%, last year it was 66.4%, and in 2017 it was 60.8%.
+
+![How would you rate your Rust expertise](/images/2020-03-RustSurvey/12-How-Rate-Rust-Expertise.png)
+
+We also asked users how they would rate their Rust Expertise - there is a clear peak around "7".
+
+## Why not use Rust?
+
+A big part of a welcoming Rust community is reaching out to non-users as well. As we have in years past, we again asked the reasons why people weren't using Rust. 
+
+![How long did you use Rust before you stopped?](/images/2020-03-RustSurvey/3-How-Long-Before-Stopped-Rust.png)
+
+For users who had used Rust before but had stopped using it, most had stopped using it within a year of starting.
+
+![Why did you stop using Rust?](images/2020-03-RustSurvey/4-Why-Stopped-Rust.png)
+
+When we asked why someone had stopped using Rust, the most common response was "My company doesn't use Rust" - which indicates Rust adoption is still the biggest reason. After that, learning curve, lack of needed libraries, being slowed down by switching to Rust, and lack of IDE support were the most common reasons a user stopped using Rust.
+
+![Why have you never used Rust?](images/2020-03-RustSurvey/5-Why-Never-Used-Rust.png)
+
+For users who indicated they had never used Rust before, most indicated either "I haven't learned Rust yet, but I want to" or "My company doesn't use Rust" - again pointing to adoption as the main hurdle.
+
+![I would use Rust more often if...](images/2020-03-RustSurvey/20-Would-use-rust-more-often-if.png)
+
+We also asked users what would lead them to use Rust more often. Most indicated they would use Rust more if their company adopted it, if Rust had more libraries that they need, and if IDE support was better. The most common reasons after those pointed to a need to improve the learning curve and interoperability. 
+
+As adoption seemed to be the biggest problem preventing some respondents from using Rust, let's dive deeper into it. 
```

Comment:
One thought -- it might make sense to change the absolute bars to percentages of total respondents to that question, or perhaps the survey in general (not sure what makes more sense).